clang/test/CXX/drs/dr14xx.cpp

```diff
@@ -153,16 +153,16 @@ namespace dr1460 { // dr1460: 3.5
   namespace Defaulted {
     union A { constexpr A() = default; };
     union B { int n; constexpr B() = default; };
-    // cxx11-17-error@-1 {{defaulted definition of default constructor is not constexpr}}
+    // cxx11-17-error@-1 {{defaulted definition of default constructor that marked constexpr but never produces a constant expression is a C++23 extension}}
```

Comment:
11 through 17 we expect a 23 extension warning, but what happens in 20 mode?

Reply:
In C++20 mode such constructor is considered as a valid and producing `constexpr` due to https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1331r2.pdf , so no extension warning there.